pytorch_lightning/trainer/training_io.py

```diff
@@ -251,10 +251,14 @@ def save_checkpoint(self, filepath):
             # do the actual save
             try:
                 self._atomic_save(checkpoint, filepath)
-            except AttributeError:
+            except AttributeError as e:
                 if 'hparams' in checkpoint:
                     del checkpoint['hparams']
 
+                m = f'warning, hparams dropped from checkpoint. An attribute ' \
+                    f'is not picklable {e}'
+                rank_zero_warn(m)
```

Comment:
I think that using `logging.exception("hparams dropped from checkpoint")` would be better